uwblueprint/jump-math
# mutation: save basic info mutation

## Notion ticket link

[Save Basic Information section on Create Assessment page](https://www.notion.so/uwblueprintexecs/Save-Basic-Information-section-on-Create-Assessment-page-76947d48c76f4e77bdff54fceaa78ea7?pvs=4)



## Implementation description
* Added a mutation for creating an assessment
* Uses the mutation when the user hits "Save"



## Steps to test
1. Go to http://localhost:3000/admin/create-assessment
2. Fill in the form and hit save
3. Check that the response object is logged
4. Go to playground and query all tests, verify that the new test exists



## What should reviewers focus on?
* 


## Checklist
